charts/katalyst/charts/agent/templates/daemonset.yaml

```diff
@@ -114,4 +118,10 @@ spec:
             path: /tmp
             type: ""
           name: host-tmp-dir
+        {{- if and (index .Values.customArgs "orm-work-mode") (eq (index .Values.customArgs "orm-work-mode") "nri") }}
```

Comment:
Also, we may need to modify the `colocation-orm` chart instead of the basic chart.